scripts/dependencies/lima.ts

```diff
@@ -207,6 +160,19 @@ export class AlpineLimaISO implements GitHubDependency {
   }
 
   rcompareVersions(version1: AlpineLimaISOVersion, version2: AlpineLimaISOVersion): -1 | 0 | 1 {
-    return rcompareVersions(version1.isoVersion, version2.isoVersion);
+    return super.rcompareVersions(version1.isoVersion, version2.isoVersion);
+  }
+
+  async updateManifest(newVersion: string | AlpineLimaISOVersion): Promise<Set<string>> {
+    if (typeof newVersion === 'string') {
+      throw new TypeError(`AlpineLimaISO.updateManifest does not support string version ${ newVersion }`);
+    }
+
+    const depVersions: DependencyVersions = yaml.parse(await fs.promises.readFile(DEP_VERSIONS_PATH, 'utf8'));
+
+    depVersions.alpineLimaISO = newVersion;
```

Comment:
Should we validate `newVersion's` structure to make sure it matches the `AlpineLimaISOVersion` interface before writing it to the YAML file?